services/explorer-indexer/src/modules/pbft/pbft.service.ts

```diff
@@ -34,6 +34,15 @@ export default class PbftService {
     this.isRedisConnected = state;
   }
 
+  public async getSavedPbftPeriods() {
+    return (
+      (await this.pbftRepository
```

Comment:
I would do this using typeorm methods like .find and .select as it is more readable.